lib/ducer/ducer.h

```diff
@@ -87,6 +87,21 @@ namespace Ducers {
   }
 
 
+  float interpolate(long rawValue, int ptType) { // 300 psi sensor
+    double values[2][2] = { // [x, y] pairs
+      {6553,  0},
+      {58982, (double) ptType}
```

Comment:
ahh GG. nicely done. 
Are we sure they all line up w/ 0.5 V = 0 psi, and 4.5V = max? 
@andycate  @taitmate38  how are we doing the new high pressure sensor? does it max at 5 or 4.5?